apis/server/container_bridge.go

```diff
@@ -105,6 +105,7 @@ func (s *Server) getContainer(ctx context.Context, rw http.ResponseWriter, req *
 		Args:            c.Args,
 		ResolvConfPath:  c.ResolvConfPath,
 		HostnamePath:    c.HostnamePath,
+		HostRootPath:    c.Snapshotter.Data["MergedDir"],
```

Comment:
Should we consider if the `c.Snapshotter.Data` may  be `nil` ?

Reply:
fixed